ios/RNTrackPlayer/Logic/MediaWrapper.swift

```diff
@@ -112,6 +112,14 @@ class MediaWrapper: AudioPlayerDelegate {
         }
     }
     
+    func updateMetadata(_ item: Track) {
+        MPNowPlayingInfoCenter.default().ap_update(
+            with: item,
+            duration: 100,
```

Comment:
we should have track duration from the item, right? perhaps even progression can be inferred here.